src/types/user.ts

```diff
@@ -518,10 +518,10 @@ export default /* GraphQL */ `
     createdAt: DateTime!
 
     """ does this address own any Travelogger NFTs? this value is cached at most 1day, and refreshed at next \`nfts\` query """
-    hasNFTs: Boolean!
+    hasNFTs: Boolean! @objectCache(maxAge: ${CACHE_TTL.LONG})
```

Comment:
this works same way as civicLiker, caching for 1 day

https://github.com/thematters/matters-server/blob/develop/src/types/user.ts#L306

```
    "Whether liker is a civic liker"
    civicLiker: Boolean! @objectCache(maxAge: ${CACHE_TTL.LONG})
```

and because the logbook icon is an overlay on top of avatar, calling rate is same as `civicLiker`,

a boolean in every response is better than querying `nfts { id }` returning array of `{ id }`s everywhere